.travis.yml

```diff
@@ -76,17 +76,14 @@ install:
     pip install --upgrade pip
     pip install --upgrade wheel
     pip install --upgrade setuptools
-  - |
-    # Install dependencies
-    if [ -z "$PRE" ]; then
-        # Install only from travis wheelhouse
-        wheelhouse_pip_install $NUMPY $PANDAS;
+  # Install only from travis wheelhouse
+  - if [ -z "$PRE" ]; then
```

Comment:
The pipe on the first line is necessary here to get proper multi-line semantics.